scripts/code_checks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Unrelated to whatever script we end up using, these tools are _slooooooow_ unless you exclude `vendor`.  PTAL at https://github.com/contiv/auth_proxy/blob/6340df9b07a1723e2419069ebb5c0d5be6850a2e/scripts/checks_in_container.sh#L10-L11

Reply:
agreed, I borrowed PKG_DIRS which already filtered out vendor
EXCLUDE_DIRS := bin docs Godeps scripts vagrant vendor install
PKG_DIRS := $(filter-out $(EXCLUDE_DIRS),$(subst /,,$(sort $(dir $(wildcard */)))))